sql/views_internal.sql

```diff
@@ -0,0 +1,28 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+
+CREATE VIEW _timescaledb_internal.missing_dimension_slices AS
```

Comment:
Hmm not a fan of this view as this is ideally always empty, cant we pull in the integrity check function from update test and make it part of this

Reply:
I agree with Sven. I don't think we should have permanent views for conditions that shouldn't occur, in particular those that should be fixed. If our fixes do the job, this view will serve no purpose.

Reply:
It was mostly intended for debugging and support purposes (either to detect or exclude a problem), which is why it was placed in the internal namespace and I think the best views for this purpose are the ones that are empty. However, if people feel it is not useful, then I'll remove it.